.gitignore

```diff
@@ -1,2 +1,5 @@
 prometheus-kemp-exporter
+.gobuild/
+bin-dist/
+build/
```

Comment:
Any files in `bin-dist/` are uploaded to the GitHub release. I've been using `build/` as a temporary directory for the uncompressed tarballs, so they don't clutter `bin-dist/`.